### ISSUE TITLE
When navigating with scan mode in Narrator, focus gets stuck on Checkbox

### ISSUE THREAD

**Issue by AndrewCraswell** [microsoft/fluentui#21025]
### Environment Information

- **Package version(s)**: `^8.22.3`
- **Browser and OS versions**: `Edge 96.0.1054.53`, `Windows 11`

### Describe the issue:

1. Navigate to the [Codesandbox](https://codesandbox.io/s/relaxed-rosalind-t99gq?file=/src/App.tsx).
2. Enable scan mode using `Caps +Spacebar`.
3. Click on the first dropdown, and use `down arrow` to navigate to the label on the checkbox.
4. Use the `up arrow` to attempt to navigate back up to the second drop down.
5. Verify that the focus gets stuck on the checkbox, and Narrator announces "Blank" with every press of the `up arrow`

#### Observation:
This only reproduces in Edge with Narrator. Chrome and Narrator, or NVDA seems to behave correctly.

### Please provide a reproduction of the issue in a codepen:
https://codesandbox.io/s/relaxed-rosalind-t99gq?file=/src/App.tsx

#### Actual behavior:

In scan mode, Narrator focus gets stuck on checkboxes while navigating in backward direction. Announces as 'Blank' with every up-arrow key stroke.

#### Expected behavior:

Narrator focus should not get stuck on checkbox and should announce correctly rather than "Blank". Instead focus should move to next elements while navigating by up/down arrow key.

### Documentation describing expected behavior
[Meaningful Sequence - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.02)

**Comment by gouttierre** [comment]
@AndrewCraswell - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley

**Comment by tringakrasniqi** [comment]
Closing this issue, for more info check: #21166
